policyengine_us/parameters/gov/states/me/tax/income/main/separate.yaml

```diff
@@ -0,0 +1,23 @@
+description: Progressive rate schedule for a separate filer in ME. 
+metadata:
+  type: marginal_rate
+  threshold_unit: currency-USD
+  rate_unit: /1
+  name: me_income_tax_rate_separate
+  label: ME State income tax rates for separate filers
+  reference:
+    - title: Maine §5111. Imposition and rate of tax
```

Comment:
Could you please add a tax form reference as well? The law doesn't specify inflation adjustments but the form would confirm it.

Reply:
Hi, Max. I'm not so familiar with the tax form. So I might be forget to search for inflation adjustment. After looking up the tax form instruction and tax rate schedules, I do find out maine performs inflation adjustments. here is an example: [the inflation adjustment for 2023](https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/ind_tax_rate_sched_2023.pdf). So basically I need to reflect those inflation adjustment by changing threshold and rate for each tax year in the yaml file?

Reply:
@MaxGhenis are you saying there's something we should add for inflation adjustments? My understanding is that Maine updates the dollar thresholds to adjust for inflation, but the thresholds we have coded in here for 2022 are adjusted and there's no formula for calculating the next period's thresholds until they release the adjustment amount.

Below is the 2023 rates as well so we could also put those in though

Tax table: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/22_1040me_tax_tables.pdf
One-Pager for main tax rates, 2022: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/ind_tax_rate_sched_2022.pdf
One-Pager for main tax rates, 2023: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/ind_tax_rate_sched_2023.pdf
Tax Form (1040): https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/22_1040me_dwnld_ff.pdf

Reply:
The numbers here match the numbers starting in 2017 in the legal code. So if they're the same that means there's no inflation adjustment. I'd suggest adding the tax form as a reference to clarify/confirm this.

Yes if 2023 rules are available please include them.

Reply:
I see what you're saying!

@mzx-4real can you add the 2022 and 2023 threshold values from the forms linked above (and might as well leave the 2017 ones in!) and add a reference to the 2023 form too?